File: SampleSetup/Output/PHP_8/Body/SomeCategory/ApiSomeCategoryOp1Request.php
<?php
namespace SampleProject\Protocol\Body\SomeCategory;

class ApiSomeCategoryOp1Request extends \SampleProject\Protocol\Body\AbstractReturn {
    public string $Op1Data;

    public function __construct()
    {
        parent::__construct();
        $this->Op1Data = "";
    }
}

File: SampleSetup/Protocol/Body/SomeCategory/ApiSomeCategoryOp1Request.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SampleProject.Protocol.Body.SomeCategory
{
    [UsingRef(nameof(AbstractReturn), "..")]
    public class ApiSomeCategoryOp1Request : AbstractReturn
    {
        public string Op1Data = "";
    }
}
